ui/appui-react/src/appui-react/layout/tool-settings/Handle.scss

```diff
@@ -4,20 +4,14 @@
  *--------------------------------------------------------------------------------------------*/
 
 .nz-toolSettings-handle {
```

Comment:
The drag handle icon is super tiny is that expected?

Reply:
@GerardasB I'll wait for you to check this change before merging.